### DISCUSSION TITLE
Return type of Symfony\Component\Console\Helper\HelperSet::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /usr/share/php/Symfony/Component/Console/Helper/HelperSet.php on line 104

### DISCUSSION THREAD

**Discussion by HustlerAnkit** [symfony/symfony#53457] [Symfony Book: The Fast Track]
Return type of Symfony\Component\Console\Helper\HelperSet::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /usr/share/php/Symfony/Component/Console/Helper/HelperSet.php on line 104

**Comment by MatTheCat** [comment]
`#[\ReturnTypeWillChange]` has been added by #42260 some years ago, and shortly after #42378 replaced it with the return type.

You should probably update your Symfony version, or downgrade your PHP version.
